### PR TITLE
Account

### PR DESCRIPTION
**Pull request by Ambre16**
added account endpoints

### CODE REVIEW

File: fill_test_data.sql [hunk 1]
```diff
@@ -1,13 +1,15 @@
+delete from exercise;
```
Comment (tim14996) [inline fill_test_data.sql:1]: Is it okay for you @vvihorev?

File: main.py [hunk 2]
```diff
@@ -53,3 +53,30 @@ def update_exercise(exercise_id: int, exercise: schemas.ExercisePatch, db: Sessi
...
     return updated_exercise
 
+@app.post("/accounts/", response_model=schemas.AccountOut)
+def create_account(account_in: schemas.AccountIn,db: Session = Depends(get_db)):
+    account_saved= crud.save_user(db,account_in)
+    return account_saved
+
+@app.get("/accounts/", response_model=list[schemas.AccountOut])
+def get_all_accounts(db: Session = Depends(get_db)):
+    accounts = db.query(models.Account).all()
```
Comment (tim14996) [inline main.py:63]: For consistency I would move it to crud.py with a own function

File: crud.py [hunk 3]
```diff
@@ -39,3 +40,37 @@ def update_exercise(db: Session, exercise_id: int, exercise: schemas.ExercisePat
...
+def get_account(db: Session, account_id: int):
+    return db.query(models.Account).filter(models.Account.id_account == account_id).first()
+
+def delete_account(db: Session, account_id: int):
+    db.delete(db.query(models.Account).filter(models.Account.id_account == account_id).first())
+    db.commit()
+
+def update_account(db: Session, account_id: int, account: schemas.AccountOut):
+    stored_account = db.query(models.Account).filter(models.Account.id_account == account_id).first()
+    if stored_account is None:
```
Comment (tim14996) [inline crud.py:69]: Remove this. See my comment on main.py

File: main.py [hunk 4]
```diff
@@ -53,3 +53,30 @@ def update_exercise(exercise_id: int, exercise: schemas.ExercisePatch, db: Sessi
...
+def delete_account(account_id: int, db: Session = Depends(get_db)):
+    db_account = crud.get_account(db, account_id=account_id)
+    if db_account is None:
+        raise HTTPException(status_code=404, detail="account not found")
+    crud.delete_account(db=db, account_id=account_id)
+    return {"message": "account deleted"}
+
+
+@app.patch("/accounts/{account_id}")
+def update_account(account_id: int, account: schemas.AccountIn, db: Session = Depends(get_db)):
```
Comment (tim14996) [inline main.py:78]: @vvihorev requested on my code that we do the none-check here in the main.py and not in the crud.py. so for consistency i would recommend to do the same here

```
    account = crud.get_account(db, account_id=account_id)
    if account is None:
        raise HTTPException(status_code=404, detail="account not found")
    changed_account= crud.update_account(db, account_id=account_id, account =account )
    return changed_account
```

**Review by vvihorev — APPROVED** [review]

**Pull request merged**